tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -394,12 +397,14 @@ export type BuckWebSocketMessage =
   | { type: \"RunComplete\" };
 
 // Missing one level of indentation because of the comment
-const rootEpic = (actions, store) =>
-  combineEpics(...epics)(actions, store).// Log errors and continue.
-  catch((err, stream) => {
-    getLogger().error(err);
-    return stream;
-  });
+const rootEpic = (actions, store) => combineEpics(...epics)(
+  actions,
+  store
+)// Log errors and continue.
```

Comment:
This expression is really hard to format... I think this has an honorable mention